activestorage/app/models/active_storage/attachment.rb

```diff
@@ -12,7 +12,7 @@ class ActiveStorage::Attachment < ActiveRecord::Base
   belongs_to :record, polymorphic: true, touch: true
   belongs_to :blob, class_name: "ActiveStorage::Blob"
 
-  delegate_missing_to :blob
+  delegate_missing_to :blob, allow_nil: true
```

Comment:
We consider it a bug for an attachment not to have a blob (see #33405), so I think we can leave this one alone.

Reply:
Sounds good, I will remove it on that one then. :+1: